translations/translations.json

```diff
@@ -8408,6 +8408,32 @@
       "showOwnLocation": "显示自己的位置"
     }
   },
+  "remoteContent": {
+    "de": {
+      "knownResourceOptIn": "Ich bin damit einverstanden, dass mir Inhalte dieser externen Quelle angezeigt werden: ",
+      "knownResourceContentMessageOne": "Dieses Element enthält Daten von ",
```

Comment:
Since the word order might be different in other languages, the variable should be included in the translation. Same below. If `knownResourceContentMessageTwo` is used directly after, they could be merged.

Reply:
Yeah missed that, sorry.